package.json

```diff
@@ -40,7 +40,8 @@
     "pg_permissions": "exec ./utils/postgraphile_launch_permissions.sh",
     "dev_pg": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg\"",
     "dev_pg_permissions": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg_permissions\"",
-    "database_init": "./database/initialise_database.sh && ./database/insert_data.sh && ./database/post_data_insert.sh",
+    "database_init": "./database/initialise_database.sh && ./database/insert_data.sh",
+    "postdatabase_init": "./database/post_data_insert.sh",
```

Comment:
When is this one being used?

Reply:
The `post` prefix means it automatically runs after `database_init` (see [here](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts). It used to be bundled in the same command (see previous diff), but the arguments were only getting passed to the last command, so needed to make them separate.